assignment-4/routes.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require('supercharge');
```

Comment:
supertest?? I'm really curious what's this package https://www.npmjs.com/package/supercharge